fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -612,7 +612,7 @@ private void initAggregateBuiltins() {
         addBuiltin(AggregateFunction.createBuiltin(BITMAP_UNION_COUNT, Lists.newArrayList(Type.BITMAP),
                 Type.BIGINT,
                 Type.BITMAP,
-                true, true, true));
+                true, false, true));
```

Comment:
bitmap union count support window function in old query engine.

I think we could support it in BE query engine.